src/plugins/dump/dump.cpp

```diff
@@ -78,125 +111,347 @@ int serialise (std::ostream & os, ckdb::Key *, ckdb::KeySet * ks)
 				/* Meta key already serialised, write out a reference to it */
 				keyDel (search);
 
-				os << "keyCopyMeta ";
-				os.write (static_cast<const char *> (ckdb::keyValue (ret)), ckdb::keyGetValueSize (ret));
-				os << std::endl;
+				os << "$copymeta " << keyString (ret);
 			}
 		}
-		os << "keyEnd" << std::endl;
-	}
-	os << "ksEnd" << std::endl;
 
+		os.flush ();
```

Comment:
Is this really needed? (os seems to be closed shortly afterwards anyway, which will do the flush)

Reply:
It probably isn't needed. I added it when trying to work out, why `pluginprocess` just hangs (which was because of the missing end marker).